### ISSUE TITLE
NTR: MRI brain: increased T1 signal in brainstem (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9013]
New term request MRI brain: increased T1 signal in brainstem (elthat_181204164922): Hyperintensities; hyperintensity; midbrain; pons; medulla

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000286
- label: Focal T1 hyperintense brainstem lesion
- parents: [Abnormal brainstem MRI signal intensity (HP:0012747)](https://hpo.jax.org/app/browse/term/HP:0012747)
- pmid: 
- def: A lighter than expected T1 signal on magnetic resonance imaging (MRI) of the brainstem. This term refers to a localized hyperintensity affecting a particular region of the brainstem.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9013
- synonyms:
